build.sbt

```diff
@@ -218,18 +214,18 @@ lazy val catsInterop = project
   )
   .dependsOn(core)
 
-lazy val monixInterop = project
-  .in(file("interop/monix"))
-  .settings(name := "caliban-monix")
-  .settings(commonSettings)
-  .settings(
-    libraryDependencies ++= Seq(
-      "dev.zio"  %% "zio-interop-reactivestreams" % zioInteropReactiveVersion,
-      "dev.zio"  %% "zio-interop-cats"            % zioInteropCats2Version,
-      "io.monix" %% "monix"                       % "3.4.1"
-    )
-  )
-  .dependsOn(core)
+//lazy val monixInterop = project
+//  .in(file("interop/monix"))
+//  .settings(name := "caliban-monix")
+//  .settings(commonSettings)
+//  .settings(
+//    libraryDependencies ++= Seq(
+//      "dev.zio"  %% "zio-interop-reactivestreams" % zioInteropReactiveVersion,
+//      "dev.zio"  %% "zio-interop-cats"            % zioInteropCats2Version,
+//      "io.monix" %% "monix"                       % "3.4.1"
+//    )
+//  )
+//  .dependsOn(core)
```

Comment:
Maybe just remove monix interop?

Reply:
There was a release of interop-cats for cats 2.x and ZIO RC6 a few days ago, I think that's all we need for monix?

Reply:
Ahh I missed that, uncommented.